libs/lb/lb-rs/src/blocking.rs

```diff
@@ -33,10 +37,34 @@ use crate::{
 #[derive(Clone)]
 pub struct Lb {
     lb: crate::Lb,
+    #[cfg(not(target_family = "wasm"))]
     rt: Arc<Runtime>,
 }
 
 impl Lb {
+    #[cfg(target_family = "wasm")]
+    pub fn init(config: Config) -> LbResult<Self> {
+        let lb = crate::Lb::init_dummy(config).unwrap();
+        Ok(Self { lb })
+    }
+
+    #[cfg(target_family = "wasm")]
+    fn block_on<F>(&self, future: F) -> F::Output
+    where
+        F: Future,
+    {
+        block_on(future)
```

Comment:
nit: I prefer to qualify instead of import a raw fn like this. There are a lot of `block_on`'s happening around here
```suggestion
        futures::executor::block_on(future)
```